internal/provider/appattributes.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2021 Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package provider
+
+import (
+	"context"
+	"terraform-provider-splunkconfig/internal/splunkconfig/config"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	appAttributesIdKey              = "app_id"
```

Comment:
Any reason for `appAttributesIdKey` and `app_id` here as opposed to `appAttributesAppIdKey`?

Reply:
Only to be consistent with my previously poorly named constants for other resources.  Sounds like I feel prey to broken windows.

Reply:
I've gone and repaired those broken windows.